### DISCUSSION TITLE
universal Hardware to start with TASMOTA

### DISCUSSION THREAD

**Discussion by darkside9009** [arendst/Tasmota#22041] [Show and tell]
Hi All,
long time ago I have created this hardware to connect different sensors. The most people don't like to solder any wires or resistors and don't have a 3d printer to make their own enclosure.
I think this is the coolest universal Hardware to use it with a lot of different sensors with Tasmota firmware, which is already pre-flashed. This board is made in Germany and to make it easy to buy, we have put it on amazon with a low price to let you play a bit with a really great TASMOTA project.
![CASmo-S1_800p](https://github.com/user-attachments/assets/ba7b4c05-27df-4fed-b63d-8fd2e6e13c43)
![CASmo-S1_V1 2](https://github.com/user-attachments/assets/1734aeac-9ff9-4632-b85a-8e1de2433d57)
![CASmo-S1_Blockschaltbild_V1 2](https://github.com/user-attachments/assets/3d2804d8-c89a-4c64-9a60-4f11925ca359)

https://casmo.info/shop/casmo-s1-2#attr=
https://www.amazon.de/Caraudio-Systems-CASmo-S1/dp/B0CH1GJXV5

**Comment by s-hadinger** [comment]
Super nice, thanks for sharing.

(I edited your last URL, it was pointing to nothing due to a MD bad syntax.)

**Comment by s-hadinger** [comment]
Oh! I see you also have a nice device for Leds: https://www.amazon.de/-/en/CASmo-WLED-CASmo-WLED-Controller/dp/B0CH1GSJJ1/

**Reply by darkside9009** [reply]
Yes its also working with WLED and Tasmota as well.

**Reply by sfromis** [reply]
The [Tasmota Supported Devices Repository](https://templates.blakadder.com/) also exists as a searchable place, including device config details, open to submissions.

**Reply by sfromis** [reply]
BTW. I noticed that Casmo-WLED is using gpio 17 for a relay. This is not ideal, as gpio 16/17 are commonly used as fixed assignments for PSRAM. To support PSRAM detection, Tasmota will normally probe for this at boot, meaning possible relay clicking.

**Reply by barbudor** [reply]
What's the difference of Casmo-WLED ?
Does it include power mosfet to directly drive leds like an H801 ?
I'm only using Ws2812 so standard gpio + 5v level shifter

One cool addition of such packaged box would be the ability to use an external antenna. The 'industrial' packaging make it nice to use on out-of-the-living-room use case where reception may be hard (garage,..)

**Reply by sfromis** [reply]
To me, the CASmo-WLED looks similar to https://templates.blakadder.com/athom_LS8P-WLED.html meaning no power transistors, but the device can be powered from a source up to 24V, which is passed to whatever addressable led strip is being used, meaning that it does not have to be 5V.

**Reply by darkside9009** [reply]
Hi,  I can create a template for Tasmota. In the next Version we will also change the GPIO17 to another PIN.

**Reply by darkside9009** [reply]
> The [Tasmota Supported Devices Repository](https://templates.blakadder.com/) also exists as a searchable place, including device config details, open to submissions.

Thank you, I have just added CASmo-S1

**Reply by Birger-T** [reply]
> > The [Tasmota Supported Devices Repository](https://templates.blakadder.com/) also exists as a searchable place, including device config details, open to submissions.
> 
> Thank you, I have just added CASmo-S1

Sorry, but I could not find CASmo in the Tasmota Supported Devices @https://www.tasmota.info/hardware/
UPDATE: nothing found here https://templates.blakadder.com/search.html

I bought a CASmo-S1 and a CASmo-WLED.
Is there any further documentation for CASmo-S1?

**Reply by darkside9009** [reply]
i will check, why its not online. You can see all information on our website casmo.info incl. firmware and Tasmota template.
If you need any help just let me know.
